src/_guides/language/sound-problems.md

```diff
@@ -240,8 +240,8 @@ For more information, see [Use proper input parameter types when overriding meth
 **Warning:**
 <code>Unsound implicit cast from <em>Class&lt;dynamic&gt;</em> to <em>Class&lt;type&gt;</em>.</code>
```

Comment:
Do you mean that it's an error, or that that message no longer appears?

If the latter, should I rewrite the https://kw-www-dartlang-1.firebaseapp.com/guides/language/sound-problems#missing-type-arguments section to talk only about the "Invalid override" error? Do I need to change the example or any of the other text?